dll/win32/shell32/wine/control.c

```diff
@@ -1173,20 +1173,68 @@ static	void	Control_DoLaunch(CPanel* panel, HWND hWnd, LPCWSTR wszCmd)
 void WINAPI Control_RunDLLW(HWND hWnd, HINSTANCE hInst, LPCWSTR cmd, DWORD nCmdShow)
 {
     CPanel	panel;
+#ifdef __REACTOS__
+    WCHAR *ptr1 = NULL;
+    WCHAR *ptr2 = NULL;
```

Comment:
```suggestion
    PWCHAR ptr1, ptr2;
```
They don't need to be pre-initialized since they get initialized again unconditionally below.

Reply:
Changed. Thanks.